loader/test/riteaid.scraper.test.js

```diff
@@ -145,7 +145,7 @@ describe("Rite Aid Scraper", () => {
         },
       });
 
-    const result = await checkAvailability(() => {}, { states: ["NJ"] });
+    const result = await checkAvailability(() => {}, { states: ["CT"] });
```

Comment:
I changed all these tests to "CT" since NJ now requests multiple zip codes. CT still only uses one, and that keeps things straightforward in the tests.